### ISSUE TITLE
PyMISP.search does not include events with 0 attributes

### ISSUE THREAD

**Issue by PabDJ** [MISP/PyMISP#1083]
I have created two events with the same tag. One has attributes and the other one is empty.

If I run:
`event_list = misp.search(collector="event", tags=my_tag)`
The list will only contain the event that was populated with attributes.

How can I retrieve the event with 0 attributes using its tag as filter?

**Comment by Rafiot** [comment]
No, the empty events aren't exposed to the search. (I'm 99% sure, but @iglocska can surely confirm)

**Comment by PabDJ** [comment]
> No, the empty events aren't exposed to the search. (I'm 99% sure, but @iglocska can surely confirm)

Is there any workaround to get those empty events with another approach?

**Comment by github-germ** [comment]
See if this helps: https://github.com/MISP/PyMISP/issues/847
